src/smc-webapp/editor.coffee

```diff
@@ -1085,8 +1085,11 @@ class CodeMirrorEditor extends FileEditor
                 that.click_edit_button($(@).data('name'))
                 return false
 
+        # FIXME: removing the button here is pointless, since it will be added later again (somewhere)
         # FUTURE: implement printing for other file types
-        if @filename.slice(@filename.length-7) != '.sagews'
+        is_sagews = @filename.slice(@filename.length-7) != '.sagews'
```

Comment:
oh well, I wasn't thinking much. before that, it was this `if @filename.slice(@filename.length-7) != '.sagews'` and that's how it is right now. I just wanted to extend this to .tex files without having a too long line. It's a bit odd, yes. Let me fix this nicely.